code/game/objects/items/theft_items.dm

```diff
@@ -0,0 +1,295 @@
+	item_state = "screwdriver_nuke"
+	toolspeed = 0.5
+	random_color = FALSE
+
+/obj/item/paper/guides/antag/nuke_instructions
+	info = "How to break into a Nanotrasen nuclear device and remove its plutonium core:<br>\
+	<ul>\
+	<li>Use a screwdriver with a very thin tip (provided) to unscrew the terminal's front panel.</li>\
+	<li>Dislodge and remove the front panel with a crowbar.</li>\
+	<li>Cut the inner metal plate with a welding tool.</li>\
+	<li>Pry off the inner plate with a crowbar to expose the radioactive core.</li>\
+	<li>Pull the core out of the nuclear device. </li>\
+	<li>Put the core in the provided container, which will take some time to seal. </li>\
+	<li>???</li>\
+	</ul>"
+
+// STEALING SUPERMATTER.
+
+/obj/item/paper/guides/antag/supermatter_sliver
+	info = "How to safely extract a supermatter sliver:<br>\
+	<ul>\
+	<li>Approach an active supermatter crystal with radiation shielded personal protective equipment. DO NOT MAKE PHYSICAL CONTACT.</li>\
+	<li>Use a supermatter scalpel (provided) to slice off a sliver of the crystal.</li>\
+	<li>Use supermatter extraction tongs (also provided) to safely pick up the sliver you sliced off.</li>\
+	<li>Physical contact of any object with the sliver will dust the object, as well as yourself.</li>\
+	<li>Use the tongs to place the sliver into the provided container, which will take some time to seal.</li>\
+	<li>Get the hell out before the crystal delaminates.</li>\
+	<li>???</li>\
+	</ul>"
+
+/obj/item/nuke_core/supermatter_sliver
+	name = "supermatter sliver"
+	desc = "A tiny, highly volatile sliver of a supermatter crystal. Do not handle without protection!"
+	icon_state = "supermatter_sliver"
+	pulseicon = "supermatter_sliver_pulse"
+
+
+/obj/item/nuke_core/supermatter_sliver/attack_tk(mob/user) // no TK dusting memes
+	return
+
+
+/obj/item/nuke_core/supermatter_sliver/can_be_pulled(user) // no drag memes
+	return FALSE
+
+/obj/item/nuke_core/supermatter_sliver/attackby(obj/item/W, mob/living/user, params)
+	if(istype(W, /obj/item/retractor/supermatter))
+		var/obj/item/retractor/supermatter/tongs = W
+		if(tongs.sliver)
+			to_chat(user, "<span class='warning'>[tongs] is already holding a supermatter sliver!</span>")
+			return FALSE
+		forceMove(tongs)
+		tongs.sliver = src
+		tongs.icon_state = "supermatter_tongs_loaded"
+		tongs.item_state = "supermatter_tongs_loaded"
+		to_chat(user, "<span class='notice'>You carefully pick up [src] with [tongs].</span>")
+	else if(istype(W, /obj/item/scalpel/supermatter) || istype(W, /obj/item/nuke_core_container/supermatter)) // we don't want it to dust
+		return
+	else
+		to_chat(user, "<span class='notice'>As it touches [src], both [src] and [W] burst into dust!</span>")
+		radiation_pulse(user, 100)
+		playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
+		qdel(W)
+		qdel(src)
+
+/obj/item/nuke_core/supermatter_sliver/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	if(!isliving(hit_atom))
```

Comment:
I'm curious as to how someone would actually be able to throw this outside of admemery, considering it just dusts you if you try to pick it up.

Reply:
Mass driver, sorium